### PR TITLE
[ES6 class] Update SidebarStore to es6 class

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
ℹ️   _This branch depends on features that will be introduced with #275, thus it will fail until the respective branch is merged._

---
Update store to es6 class – this is part of series of PRs to update all the stores as discussed in https://github.com/dcos/dcos-ui/pull/246/files/b4b52fcdb5f899fd74f9bd68cfa3d216177fcecf#r64962494

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/stores/SidebarStore.js [hunk 1]
```diff
@@ -1,72 +1,74 @@
...
+          this.emitChange(EventTypes.SHOW_VERSIONS_ERROR);
+          break;
+        case ActionTypes.SIDEBAR_WIDTH_CHANGE:
+          this.emitChange(EventTypes.SIDEBAR_WIDTH_CHANGE);
+          break;
+      }
 
-  mixins: [GetSetMixin],
+      return true;
+    })
```
Comment (mlunoe) [inline src/js/stores/SidebarStore.js]: #nit missing semi-colon

File: src/js/stores/SidebarStore.js [hunk 2]
```diff
@@ -1,72 +1,74 @@
...
+  init() {
     this.set({
       isOpen: false,
       versions: {}
     });
-  },
+  }
 
-  emitChange: function (eventName) {
+  emitChange(eventName) {
```
Comment (mlunoe) [inline src/js/stores/SidebarStore.js]: This was not you, but we should probably remove this function. Did you create a ticket for it?

Comment (mlunoe) [inline src/js/stores/SidebarStore.js]: Maybe we can add the same comment here: `// TODO: DCOS-7430 - Remove emitChange method`

**Pull request merged**